wrappers/python/examples/llama_training.py

```diff
@@ -12,89 +12,79 @@
 # @version 1.0.0
 
 import argparse
-# import json
+import json
 import time
 
 import numpy as np
 import torch
-# import torch.nn as nn
-# from datasets import load_dataset
-# from torch.optim import SGD, Adam, AdamW
-from transformers import LlamaConfig, LlamaModel
+from torch.optim import SGD, Adam, AdamW
+from transformers import LlamaConfig, LlamaForCausalLM, LlamaTokenizerFast
 
 import nntile
-# from nntile.layer.llama_mlp import LlamaMLP as LlamaMLP_nntile
-# from nntile.loss import Frob
-from nntile.model.llama import Llama as Llama_nntile, LlamaConfigNNTile
-
-# from transformers import LlamaTokenizerFast
-
+from nntile.model.llama_causal import LlamaForCausalLM as Llama_nntile
+from nntile.model.llama_config import LlamaConfigNNTile
 
 # Create argument parser
-parser = argparse.ArgumentParser(prog="Llama-based neural networks",
+parser = argparse.ArgumentParser(prog="LLaMa-based neural networks",
         description="This example presents an NNTile implementation of a "
-        "Llama-family of models and compares it against the Huggingface. "
+        "LLaMa-family of models and compares it against the Huggingface. "
         "It checks relative accuracy of a forward pass (values of "
         "activations) and backward pass (gradients of parameters and "
         "activations) and a throughput of inference and training. It can "
         "also fine-tune a pretrained NNTile model on a chosen dataset.")
-parser.add_argument("--model", default="llama")
+
+# parser.add_argument("--remote_model_name", default="huggyllama/llama-7b")
```

Comment:
are there any smaller versions of llama available? Maybe put here one of the smallest?